ONSdigital/response-operations-ui
# Group vacancies messages

# Motivation and Context
Vacancies, like FDI, needed to have the messages around it grouped together.  Most surveys have a 1 to 1 mapping of survey_id to survey.  Vacancies has 4 survey ids that all relate to the vacancies survey as a whole.

# What has changed
The code that groups the 4 FDI surveys has been copied for vacancies.

# How to test?
- Run acceptance tests to seed some data (This needs to be on the `setup-vacancies` branch)
- Open response ops and go to the messages tab.
- Take a look at the options for surveys, one of them should be 'Vacancies'
- If you click on that,  you'll see it empty of messages (this is fine).  Look at your logs and you'll see a call made to secure message that looks similar to:
```
http://localhost:5050 "GET /threads?survey=07444b92-e8e4-4f65-902c-b630a0e0728b&survey=1107f76d-9de3-4a05-87f4-043a55fe29ec&survey=21976698-c4ee-4c06-bc81-92ed3ff0d85b&survey=4ebca142-fe5d-407c-82e3-235b6c9bb428&is_closed=false&my_conversations=false&new_respondent_conversations=false&all_conversation_types=false&page=1&limit=10 HTTP/1.1" 200 397
```
The important bit is the multiple survey ids in the call (if you're interested, these surveys match the ids that are in the survey services database)

# Links
https://trello.com/c/botlRPsE

# Screenshots (if appropriate):
